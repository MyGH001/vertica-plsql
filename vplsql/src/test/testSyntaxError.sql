/**
* testing error message with incorrect syntax PL/SQL.
*/

select PLSQL_CREATE(using parameters content=
$$create or replace procedure p_test_syntax_error()
as
end;$$);

select PLSQL_EXEC(
$$create or replace procedure p_test_syntax_error()
as
end;$$ using parameters withStderr=true);
