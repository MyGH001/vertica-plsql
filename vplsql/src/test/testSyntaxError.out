PL/SQL syntax error! Details:
line 3:0 mismatched input 'end' expecting <EOF>
Parser tree: (program (block (stmt (create_procedure_stmt create or replace procedure (ident p_test_syntax_error) (create_routine_params ( )) (proc_block (stmt (expr_stmt (expr (expr_atom (ident (non_reserved_words as)))))))))) end ;)
line 3:0 mismatched input 'end' expecting <EOF>
